sdk/python/tests/run_test_server.py

```diff
@@ -452,12 +453,16 @@ def run_nginx():
 
     env = os.environ.copy()
     env['PATH'] = env['PATH']+':/sbin:/usr/sbin:/usr/local/sbin'
+    os.mkfifo(nginxconf['ACCESSLOG'], 0700)
```

Comment:
Because `TEST_TMPDIR` is static, this needs to deal with the fact that the FIFO might already exist.  Otherwise this happens on subsequent test runs:

```
Traceback (most recent call last):
  File "./run_test_server.py", line 628, in <module>
    run_nginx()
  File "./run_test_server.py", line 456, in run_nginx
    os.mkfifo(nginxconf['ACCESSLOG'], 0700)
OSError: [Errno 17] File exists
rake aborted!
Command failed (pid 7530 exit 1): [["python", "./run_test_server.py", "start_nginx"]]
/home/brett/repos/arvados/apps/workbench/test/test_helper.rb:165:in `block in check_output'
…
```

Everything else looks good to me.